libtransmission/transmission.h

```diff
@@ -999,6 +999,7 @@ bool tr_torrentSetMetainfoFromFile(tr_torrent* torrent, tr_torrent_metainfo cons
 char const* tr_torrentName(tr_torrent const*);
 
 uint64_t tr_torrentTotalSize(tr_torrent const*);
```

Comment:
Is new API necessary? I think this is already available in tr_stat.sizeWhenDone

Reply:
We're querying that value for all torrents (you may have more than 1000) every second or less.
We don't need the whole `stat` object, just the sizeWhenDone.

Reply:
But doesn’t Torrent.mm already have the strat cached in fStat? No need for a new query?